utilities/db-migration/migration_test/add_test_values/add_db_values.go

```diff
@@ -0,0 +1,150 @@
+package addtestvalues
+
+import (
+	"github.com/redhat-appstudio/managed-gitops/backend-shared/db"
+)
+
+// To avoid code duplication all the structs are declared as variable which can be acessed in our test files
+var (
+	AddTest_PreClusterUser = db.ClusterUser{
+		Clusteruser_id: "test-user-1",
+		User_name:      "test-user-1",
+	}
+	clusterUser = AddTest_PreClusterUser
```

Comment:
Looks great! A final tweak, can we remove the lowercase variables from the variable list (e.g. `clusterUser`, `clusterCredentials`, etc), and just reference these variables via the `AddTest_*` version of these tests?

My guess is that you are splitting them in two so that in the add tests you can modify the lowercase version of the variable? (Correct me if I'm wrong)

I think you can instead do this, in the add tests:
```go
			By("Create cluster user")
			clusterUser := AddTest_PreClusterUser
			err = dbq.CreateClusterUser(ctx, &clusterUser)
			Expect(err).To(BeNil())
```

This should allow the test to only modify the local variable data, without worrying about the Create function modifying the constant.